backend/lib/rpc-service/service.js

```diff
@@ -14,9 +14,13 @@ import { ClaimEvent, RawActorEvent, BlockEvent, RpcRespone } from './data-types.
  */
 export const rpcRequest = async (method, params) => {
   const reqBody = JSON.stringify({ method, params, id: 1, jsonrpc: '2.0' })
+  const headers = { 'content-type': 'application/json' }
+  if (rpcHeaders.Authorization) {
+    Object.assign(headers, rpcHeaders)
+  }
```

Comment:
This can be simplified as follows:

```suggestion
  const headers = { 
    ...rpcHeaders,
    'content-type': 'application/json'
  }
```

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Also note that header names are case-insensitive (at least in JavaScript), `rpcHeaders.authorization` and `rpcHeaders.Authorization` works the same way, so it's best not to rely on a specific property name here, and use the spread syntax instead.